apps/condo/lang/ru/ru.json

```diff
@@ -1426,23 +1426,34 @@
   "news.filed.shareNews.button": "Отправить новость",
   "news.initialOrganizationNewsItem.body": "Управляющая компания [название УК] рада приветствовать вас в приложении Doma!\nКаждый день мы работаем, чтобы ваша жизнь в доме была комфортной. Наши сотрудники готовы оперативно решать любые вопросы: о содержании дома, придомовой территории и работе управляющей компании.\n\nКак связаться с нами:\n1. Отправить заявку из мобильного приложения: сделать это можно в разделе «Заявки».\n2. Набрать номер аварийно-диспетчерской службы: +7 000 000 0000 — работает круглосуточно.\n3. Позвонить в офис управляющей компании: +7 000 000 0000  — работаем [РЕЖИМ РАБОТЫ].\n4. Прийти в офис управляющей компании: [АДРЕС И РЕЖИМ РАБОТЫ ОФИСА].\n\nВ приложении мы будем делиться с вами важными новостями. Спасибо, что выбрали нас ❤️",
   "news.initialOrganizationNewsItem.title": "Письмо управляющей компании 📩",
-  "news.template.category.cleaning": "\uD83E\uDDF9 Уборка",
-  "news.template.category.electricity": "\uD83D\uDCA1 Электроэнергия",
-  "news.template.category.electricityAndElevators": "\uD83D\uDCA1 Электроэнергия и лифты",
-  "news.template.category.elevators": "\uD83D\uDED7 Лифты",
-  "news.template.category.gas": "♨\uFE0F  Газ",
-  "news.template.category.heating": "\uD83C\uDF21\uFE0F Отопление",
-  "news.template.category.intercom": "\uD83D\uDD22 Домофон",
-  "news.template.category.meters": "⏱\uFE0F Счетчики",
-  "news.template.category.other": "\uD83C\uDFE0 Другое",
-  "news.template.category.snow": "❄\uFE0F Снег",
-  "news.template.category.telephone": "\uD83D\uDCDE Телефон",
-  "news.template.category.utilities": "\uD83D\uDCA1 Все коммунальные услуги",
-  "news.template.category.water": "\uD83D\uDCA7 Вода",
-  "news.template.category.fireProtection": "\uD83D\uDC68\uD83C\uDFFB Противопожарное",
+  "news.template.category.cleaning": "🧹 Уборка",
+  "news.template.category.electricity": "💡 Электроэнергия",
+  "news.template.category.electricityAndElevators": "💡 Электроэнергия и лифты",
+  "news.template.category.elevators": "🛗 Лифты",
+  "news.template.category.fireProtection": "👨🏻 Противопожарное",
+  "news.template.category.gas": "♨️  Газ",
+  "news.template.category.heating": "🌡️ Отопление",
+  "news.template.category.intercom": "🔢 Домофон",
+  "news.template.category.meters": "⏱️ Счетчики",
+  "news.template.category.other": "🏠 Другое",
+  "news.template.category.snow": "❄️ Снег",
+  "news.template.category.telephone": "📞 Телефон",
+  "news.template.category.utilities": "💡 Все коммунальные услуги",
+  "news.template.category.water": "💧 Вода",
```

Comment:
This is something that @Alexander-Turkin changed in recent PR

Reply:
@Alexander-Turkin ^^

Reply:
just ran `yarn workspace @app/condo lint-translations --fix`